VideoWeb/VideoWeb/ClientApp/src/app/shared/self-test/self-test.component.spec.ts

```diff
@@ -434,11 +434,14 @@ describe('SelfTestComponent', () => {
             // Arrange
             const disconnectSpy = spyOn(component, 'disconnect');
             const callSpy = spyOn(component, 'call');
+            const stream = new MediaStream();
+            component.preferredMicrophoneStream = stream;
 
             // Act
             component.replayVideo();
```

Comment:
This currently doesn't test that it gets the active microphone from the user media service.